src/sentry/static/sentry/app/components/editableText.tsx

```diff
@@ -0,0 +1,189 @@
+// eslint-disable-next-line sentry/no-react-hooks
```

Comment:
we still have to enable hooks :)

Reply:
Wondering if it's worth to remove this eslint rule now?

Reply:
it was just merged 🥳    Mark is working on a new release

Reply:
Release is out, and I'll have `master` updated soon.